src/screens/FeedScreen/Content.js

```diff
@@ -433,10 +449,10 @@ export const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
     height: '100%',
-    backgroundColor: `${bg}`
+    backgroundColor: COLORS.transparent
   }),
   centerVerticalText: () => ({
-    color: 'black',
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundColor` property in `centerVertical` style and `color` property in `centerVerticalText` style have been changed. Make sure these changes align with your design requirements.
```diff
+ backgroundColor: COLORS.transparent
...
+ color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->